blocks/library/image/style.scss

```diff
@@ -15,6 +15,11 @@
 		float: right;
 		margin-left: 15px;
 	}
+
+	&[data-align="full"] {
+		width: 90%;
+		max-width: none;
+	}
```

Comment:
Should we center the image as well?

```css
&[data-align="full"] {
    img {
        margin: 0 auto;
    }
}
```

Not a huge fan of over nesting selectors but YOLO!